spec/services/broadcasts/welcome_notification/generator_spec.rb

```diff
@@ -96,31 +92,52 @@
       expect(Notification).not_to have_received(:send_welcome_notification).with(user.id, github_connect_broadcast.id)
     end
 
-    it "does not send a duplicate notifications (github)" do
-      allow(Notification).to receive(:send_welcome_notification).and_call_original
-
+    it "does not send duplicate notifications (github)" do
       user = create(:user, :with_identity, identities: ["twitter"], created_at: 1.day.ago)
       2.times do
         sidekiq_perform_enqueued_jobs { described_class.new(user.id).send(:send_authentication_notification) }
       end
+      expect(user.notifications.count).to eq(1)
```

Comment:
I didn't apply `.and_call_original` to these specs originally because I wanted to avoid additional database calls. I wanted to leave that responsibility to `notification_spec`.  I suppose this change gives better coverage but isn't it a bit excessive to still keep the expectation below (line 101)?

Reply:
Yeah, I pulled it out because I wanted to rely on the that in my tests, too, and it seemed like the right time to do it. But looking at again now after reading your comment, I think that you're right that the double assertion here is a bit overkill. I'll remove references to it in these tests since we're relying on that and just check for `notifications.count.